### PR TITLE
s/member/participant, and other light edits

### LINKED ISSUE #1: Is membership clients or users? Pick one
We will tie ourselves in knots if we use the word member and membership to mean both the clients and the users who interact with a room.

I propose we reserve the term `member` to be for clients. For users, I propose either "a user in the room", or a "participant" in the room.

### LINKED ISSUE #8: Transport does not describe events
https://bifurcation.github.io/mimi-arch/draft-barnes-mimi-arch.html#section-5.1-5

> The MIMI transport protocol defines this event framework, including its authentication scheme, as well as the mechanics of how events are delivered from one server to another.

The signaling protocol defines how rooms actually operate, encompassing the user-level membership control protocol, half of the policy control protocol, and enforcement of that policy. To be effective in its responsibilities, the event schema and framework needs to be specified at a higher level than the transport, ideally within the signaling protocol itself. 

The transport absolutely needs to consider how to authenticate traffic between servers, but it does not have ownership of the event structure itself. A signaling protocol instead describes what constitutes a legal event under the policy. A transport protocol may however serialize events in a different format than used by signaling.

In other words, integrity and authorization of events is not a concern for the transport, but rather signaling. The transport will still need to be aware of what events are and how they work to make its own decisions on. For example, only returning events the requesting server has visibility on.

### LINKED ISSUE #9: "membership control protocol" is now presumably "participation control protocol"

### PR DESCRIPTION
**Pull request by turt2live**
See commits.

Fixes https://github.com/bifurcation/mimi-arch/issues/8
Fixes https://github.com/bifurcation/mimi-arch/issues/9
Fixes https://github.com/bifurcation/mimi-arch/issues/1

### CODE REVIEW

File: draft-barnes-mimi-arch.md [hunk 1]
```diff
@@ -132,18 +133,9 @@ the scope of MIMI.  The hub server establishes the initial state of the room.
...
 * The authorization policy for the room
 
-> NOTE: We use the phrase "membership" for both users and clients.  It might be
-> clearer to invent some distinct terminology for these notions.
-
-> NOTE: In the below, we discuss user-level membership as something independent
-> from, and managed separately from, the end-to-end security state of the room.
-> Another possibility is that the user-level membership could be derived from
-> the client-level membership, in the sense that a user is a member if and only
-> if they have one or more clients joined.
```
Comment (turt2live) [inline draft-barnes-mimi-arch.md:145]: This is removed because of formed consensus within the design team

File: draft-barnes-mimi-arch.md [hunk 2]
```diff
@@ -71,12 +71,13 @@ to a single user.  Each provider is represented by a _server_ (logically a
...
-terms of both users and clients) and policies about things like how the room may
-be joined and what capabilities each member has.
+take place in the context of a room.  Rooms have a notion of
+_user participation_ as well as _client membership_, both tracked as lists.
+Rooms additionally have policies about things like how the room may be joined
+and what capabilities each member/participant has.
 
 The protocol interactions that drive a room unfold among the servers whose users
-are members of the room.  There is exactly one _hub_ server for the room, which
+are participants of the room.  There is exactly one _hub_ server for the room, which
```
Comment (bifurcation) [inline draft-barnes-mimi-arch.md]: I would use "participants in" vs. "participants of".

File: draft-barnes-mimi-arch.md [hunk 3]
```diff
@@ -161,32 +153,33 @@ identities and credentials presented by the clients in the room.  The private
...
 the clients.
 
-## Membership
+## Participation and Membership
 
-The membership of a room is the set of users who are allowed to participate in
-the room in some way.  The specific list of ways in which a user may
-participate is defined by authorization policy, as discussed in {{policy}}.
+Users are considered to be _participating_ in a room while clients of those
+users are _members_ of the end-to-end security state for the room.  The specific
```
Comment (bifurcation) [inline draft-barnes-mimi-arch.md]: The "while" here reads ambiguously -- I initially parsed it as meaning that a user is only a participant if they have clients joined.  Suggest the following rewrite:

```suggestion
For clarity, we use different words to express the inclusion of users and clients 
in a room.  Users are `participants` in a room; clients are `members` of the
end-to-end security state of the room.  The specific
```

File: draft-barnes-mimi-arch.md [hunk 4]
```diff
@@ -161,32 +153,33 @@ identities and credentials presented by the clients in the room.  The private
...
 other actions.  It is up to client implementations how this state is
 represented) The reverse situation is forbidden; a client belonging to a user
 may be joined to the end-to-end security state of the room only if its user is a
-member.
+joined participant of the room.
 
 ## Membership Changes
 
-The membership of a group can change over time, via _add_ and _remove_
+The collective membership of a group can change over time, via _add_ and _remove_
```
Comment (bifurcation) [inline draft-barnes-mimi-arch.md]:
```suggestion
The participant list and client membership of a group can change over time, via _add_ and _remove_
```

**Review by bifurcation — CHANGES_REQUESTED** [review]

File: draft-barnes-mimi-arch.md (hunk 2 above)
Comment (turt2live) [reply draft-barnes-mimi-arch.md]:
```suggestion
are participants in the room.  There is exactly one _hub_ server for the room, which
```

**Review by bifurcation — APPROVED** [review]

**Pull request merged**